service/definition/service.go

```diff
@@ -52,7 +56,7 @@ type Dependency struct {
 	Volumes []string `yaml:"volumes" json:"volumes,omitempty" validate:"unique,dive,printascii"`
 
 	// VolumesFrom are the docker volumes-from from.
-	VolumesFrom []string `yaml:"volumesfrom" json:"volumesFrom,omitempty" validate:"unique,dive,printascii"`
+	VolumesFrom []string `yaml:"volumesFrom" json:"volumesFrom,omitempty" validate:"unique,dive,printascii"`
```

Comment:
```suggestion
	VolumesFrom []string `yaml:"volumesfrom" json:"volumesFrom,omitempty" validate:"unique,dive,printascii"`
```

This is not a typo (or was?) and should not be changed in this PR because it's a breaking change.

Reply:
Done in https://github.com/mesg-foundation/core/pull/824